### PR TITLE
Coding Challenge 153 - Interactive Drawing with SketchRNN

### PR DESCRIPTION
**Pull request by violetcraze**
Adding page for coding challenge 153.

@shiffman Can you double check the code and header formatting to make sure it looks good?

### CODE REVIEW

File: CodingChallenges/CC_153_Interactive_SketchRRN/P5/rdp.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+let epsilon = 5;
```
Comment (shiffman) [inline CodingChallenges/CC_153_Interactive_SketchRRN/P5/rdp.js]:
```suggestion
// The Coding Train
// https://thecodingtrain.com/CodingChallenges/153-interactive-sketchrnn.html
// https://youtu.be/ZCXkvwLxBrA
// https://editor.p5js.org/codingtrain/sketches/hcumr-aua

let epsilon = 5;
```

Should the headers say "The Coding Train" instead of or in addition to "Daniel Shiffman"?

File: CodingChallenges/CC_153_Interactive_SketchRRN/P5/sketch.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  if (personDrawing) {
+    // let strokePath = {
+    //   dx: mouseX - pmouseX,
+    //   dy: mouseY - pmouseY,
+    //   pen: 'down'
+    // }
+    // line(x, y, x + strokePath.dx, y + strokePath.dy);
+    // x += strokePath.dx;
+    // y += strokePath.dy;
+    // seedPath.push(strokePath);
```
Comment (shiffman) [inline CodingChallenges/CC_153_Interactive_SketchRRN/P5/sketch.js]: I think it's ok to clean up the code a bit and remove things that I commented out in the livestream but didn't delete. I might keep them in sometimes if they are meaningful but in this case I think it could go?

File: _CodingChallenges/153-interactive-sketchrnn.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+videos:
+  - title: "Challenge #128: Sketch-RNN Snowflakes"
+    author: "The Coding Train"
+    url: "/CodingChallenges/128-sketchrnn-snowflakes"
+  - title: "Challenge #152: RDP Algorithm"
+    author: "The Coding Train"
+    url: "/CodingChallenges/152-rdp-algorithm"
+---
+
+This challenge uses SketchRNN from ml5.js to create a program where the person starts the drawing and computer will finish it using whatever SketchRNN model is loaded.
```
Comment (shiffman) [inline _CodingChallenges/153-interactive-sketchrnn.md]:
```suggestion
This challenge uses  the pre-trained SketchRNN machine learning model (available with the ml5.js library) to create a p5.js sketch that finishes a person's drawing.
```

Minor changes, thank you for the description! I try not to be too wordy, but I added some stuff here which is often good for YouTube search keywords.

File: CodingChallenges/CC_153_Interactive_SketchRRN/P5/sketch.js (hunk 2 above)
Comment (violetcraze) [reply CodingChallenges/CC_153_Interactive_SketchRRN/P5/sketch.js]: Good to know. Removed things that didn't seem like they would be useful if they were uncommented
